src/components/BuildsList/BuildsList.tsx

```diff
@@ -181,7 +181,7 @@ export const BuildsList = ({ serviceContainerBuilds, columns = defaultColumns, c
               id={StorageKeys.isBuildsListCompactMode}
               label="Compact Mode"
               isChecked={isCompactMode}
-              onChange={(checked) => {
+              onChange={(_, checked) => {
```

Comment:
Out of curiosity, are you saying by "_" that there is no event?

Reply:
the event is there, `_` or string prepended by `_`, is a convention to name variables to denote that the value is not being used (linter if properly set complains the variable is declared but unused)